src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Altinn.Platform.Storage.Interface.csproj

```diff
@@ -24,7 +24,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.11" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="5.0.2" />
```

Comment:
We need to roll back this as well.

Reply:
We need to be careful here. Having this project reference a .net core 3.1 dependency and then having the storage project using a .net 5 version of the same package. I hope it will work, but we should take extra steps to make sure.